src/fileWriter.ts

```diff
@@ -107,28 +114,33 @@ const verificationKeyType = (path: BIP32Path): string => {
 
 const verificationKeyDescription = (path: BIP32Path): string => {
   switch (classifyPath(path)) {
-    case PathTypes.PATH_POOL_COLD_KEY:
-      return 'Stake Pool Operator Verification Key'
+    case PathTypes.PATH_WALLET_SPENDING_KEY_BYRON:
+      return 'Hardware Payment Verification Key'
 
     case PathTypes.PATH_WALLET_SPENDING_KEY_SHELLEY:
-      return 'Payment Verification Key'
+      return 'Hardware Payment Verification Key'
```

Comment:
In the 3rd party tools like my SPO Scripts, Jormanager, the CN Tools, etc... we add the word Hardware in after we created the keys. Because we need to know if the Verification Key belongs to a Hardware Signing File, or a CLI Signing File.

Reply:
I can't decide where to put the "Hardware"... 
I put it in the beginning because I think some checks are easier that way (e.g. one can search for "Payment Verification Key" as a continuous substring).
What is the preferred way to do it?  @gitmachtl

Reply:
Its up to you, as far as i know the tools are just searching for the keyword "Hardware" in the description. Not importen where it appears in the description string.